index.html

```diff
@@ -87,5 +95,52 @@
         hidePreloader();
         });
         </script>
-    </body>
+
+    <!---------------------------------->
+
+
+    <!-----------Main Body--------->
+    <div id="video">
+      <video autoplay loop>
+        <source src="assets/video/homepage/doc2pen video.m4v" type="video/mp4">
```

Comment:
m4v is more preferrable for Apple Devices and if you check the deploy at https://deploy-preview-31--doc2pen.netlify.app/, it's neither playing on a windows system or Ubuntu System

Reply:
it's playing on mine